src/app/modules/auth-private/resendVerification/resendVerification.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from 'src/app/shared/services/user.service';
+import { Apollo } from 'apollo-angular';
+import gql from 'graphql-tag';
+import { ErrorService } from 'src/app/shared/components/error-modal/error.service';
+
+const ResendEmailConfirmationMutation = gql`
+mutation resendEmail($email: String!) {
+  authResendVerificationEmail(email: $email)
+}
+`;
+
+@Component({
+  selector: 'app-resendVerification',
+  templateUrl: './resendVerification.component.html',
+  styleUrls: ['./resendVerification.component.scss']
+})
+export class ResendVerificationComponent implements OnInit {
+  currentUser: any;
+
+  constructor(
+    private userService: UserService,
+    private apollo: Apollo,
+    private errorService: ErrorService
+    ) { }
+
+  ngOnInit() {
+    this.getCurrentUser();
+  }
+  resendEmailVerification() {
+    const variables = {
+      email: this.currentUser.email
+    };
+    this.apollo.mutate({mutation: ResendEmailConfirmationMutation, variables: variables})
+    .subscribe ((res) => {
+      if (res) {
```

Comment:
@Harshwardhan18 You should notify the user in the template file once the Verification email has been sent

Reply:
@avikt  bro check this commit here I've removed all the unwanted code lines